src/resources/Enums.ts

```diff
@@ -231,6 +231,11 @@ export enum FilterStatusType {
     ERROR = 'ERROR',
 }
 
+export enum StringFilterType {
```

Comment:
For me `StringFilterType` could be related to any ressource in the platform-client, I think the name is way too generic. 

That being said, I'm not a big fan of enums for those case and I think we should use string literal instead. See my suggestion in the other file